tools/renovate/otter-project.json

```
@@ -0,0 +1,86 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base"
+  ],
+  "labels": [
+    "upgrade",
+    "otter"
+  ],
+  "updateInternalDeps": true,
+  "automerge": true,
+  "platformAutomerge": true,
+  "enabledManagers": [
+    "npm"
+  ],
+  "npm": {
+    "stabilityDays": 0
+  },
+  "packageRules": [
+    {
+      "matchManagers": [
+        "yarn"
+      ],
+      "matchPackageNames": [
+        "@o3r/core",
+        "@ama-sdk/core",
+        "@otter/core"
+      ],
+      "postUpgradeTasks": {
+        "commands": [
+          "yarn install",
+          "yarn run ng update {{{depName}}} --from={{{currentVersion}}} --to={{{newVersion}}} --migrate-only --allow-dirty --force"
+        ],
+        "fileFilters": [
+          "!**/.npmrc"
```

If we need to patch the yarnrc to run the install and subsequent script, the yarnrc should also be excluded from the files to commit
```suggestion
          "!**/.{npmrc,yarnrc*}"
```